### PR TITLE
Remove canvas minimap when viewing fns

### LINKED ISSUE #4006: UI freezes when going to a function
Related to large or small traces perhaps. Or maybe only when coming from a handler. Haven't got all the conditions down yet.

### PR DESCRIPTION
**Pull request by StachuDotNet**
Will resolve #4006.

### CODE REVIEW

**Comment by pbiggar** [comment]
Gave it a quick look, and directionally this looks right.

File: client/styles/_canvas.scss [hunk 1]
```diff
@@ -230,11 +230,10 @@ body #app * {
...
       color: $cyan;
     }
-    img {
-      background-color: $grey1;
-      border: $cyan 2px solid;
-      width: 200px;
+    a {
+      width: 110px;
       height: auto;
+      bottom: 10px;
```
Comment (StachuDotNet) [inline client/styles/_canvas.scss:236]: does nothing; remember how to CSS

File: client/styles/_canvas.scss [hunk 2]
```diff
@@ -208,16 +208,16 @@ body #app * {
...
-  width: 260px;
+  width: 170px;
   height: auto;
   max-height: 300px;
 
   cursor: pointer;
   text-align: center;
   color: $cyan;
-  font-size: 10px;
+  font-size: 14px;
```
Comment (StachuDotNet) [inline client/styles/_canvas.scss:220]: todo: figure out a standard font size, or more likely use the same font as the docs icon next to it

File: client/src/app/Main.res [hunk 3]
```diff
@@ -1975,7 +1970,7 @@ let update_ = (msg: msg, m: model): modification => {
   | GoToArchitecturalView =>
     Many(list{
       ReplaceAllModificationsWithThisOne(
-        m => ({...m, canvasProps: {...m.canvasProps, minimap: None}}, Cmd.none),
+        m => ({...m, canvasProps: m.canvasProps,}, Cmd.none),
```
Comment (pbiggar) [inline client/src/app/Main.res:1973]:
```suggestion
        m => (m, Cmd.none),
```

File: client/styles/_canvas.scss [hunk 4]
```diff
@@ -207,34 +207,31 @@ body #app * {
 
 .minimap {
```
Comment (pbiggar) [inline client/styles/_canvas.scss:208]: Is there a reason for the minimal class to exist anymore? If so, it should be renamed to something else?

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks good but I don't understand the CSS changes. Can you go though them (or remove the css if it's not needed anymore)?

File: client/src/app/Main.res (hunk 3 above)
Comment (StachuDotNet) [reply client/src/app/Main.res:1973]: Ah, good point.

That leaves the code like this:
```
| GoToArchitecturalView =>
    Many(list{
      ReplaceAllModificationsWithThisOne(
        m => (m, Cmd.none),
      ),
      Deselect,
      MakeCmd(Url.navigateTo(Architecture)),
    })
```

It seems like I can remove the `ReplaceAllModificationsWithThisOne` element of that list, then?

Comment (StachuDotNet) [reply client/src/app/Main.res:1973]: Maybe this is wrong - could be some sort of 'reset pending modifications' that I don't understand.

- [ ] figure out what `ReplaceAllModificationsWithThisOne` is - it seems a bit scattered in code.

Comment (StachuDotNet) [reply client/src/app/Main.res:1973]: after more inspection, I'm going to remove it - seems useless.

**Comment by StachuDotNet** [comment]
Renamed `minimap` to `back-to-canvas`.

The remaining styling, with commentary:
```
.back-to-canvas {
  // positioning
  position: fixed;
  bottom: 4px;
  right: 65px;
  width: 170px;

  // styling to be consistent with the UI elements and surrounding features (docs link)
  cursor: pointer;
  text-align: center;
  color: $cyan;
  font-size: 16px;

  .back-to-canvas-content {
    // for some centering of the text and icon
    display: flex;
    align-items: center;
    justify-content: space-between;

    .help-icon {
      font-size: 45px;
      color: $cyan;
    }
    a {
      width: 110px;
    }
  }

  .tooltips {
    // otherwise, the tooltip is too far to the right of the ?
    right: 23px;
  }
}
```

**Comment by StachuDotNet** [comment]
I just experienced this lag in prod, finally - wow! bad.

**Review by pbiggar — APPROVED** [review]

**Pull request merged**